### DISCUSSION TITLE
New slack user cant sign in

### DISCUSSION THREAD

**Discussion by Casburggraaf** [outline/outline#3549] [Self Hosting]
We are using outline as a wiki for our compony. Only since a few weeks new members cant sign-in/up with slack. Users create before can sign in without problem. It just hangs on the allow button
![afbeelding](https://user-images.githubusercontent.com/373753/168846716-c06e85ae-e540-46e0-bbe8-409b972eb621.png)

**Comment by Casburggraaf** [comment]
any ideas?

**Reply by tommoor** [reply]
No ideas. You should post the logs from your server, perhaps it isn't able to contact Slack to complete the authentication?

**Reply by Casburggraaf** [reply]
Problem is only with new registrations. 
The new members are visible on in settings members as invited but can't login.
Old members can login with slack oath.

I don't see anything in the logs

**Reply by Casburggraaf** [reply]
@tommoor Is there a way to debug this better. I saw parameters for better logging for email sending. Can i enable something for slack logging aswell?

**Reply by tommoor** [reply]
All debugging is output to stdout/stderr. Because it's hanging in this case it likely means a required API request is taking a long time – the only API request here would be one to to the Slack API though.

I'd expect an error to be output after 30 seconds or so (timeout).

**Reply by Casburggraaf** [reply]
i found an error in the logs. Can you see maybe what is wrong?
`outline  | {"stack":"AuthorizationError\n    at Strategy.OAuth2Strategy.authenticate (/opt/outline/node_modules/passport-oauth2/lib/strategy.js:138:25)\n    at attempt (/opt/outline/node_modules/passport/lib/middleware/authenticate.js:366:16)\n    at authenticate (/opt/outline/node_modules/passport/lib/middleware/authenticate.js:367:7)\n    at /opt/outline/node_modules/@outlinewiki/koa-passport/lib/framework/koa.js:194:7\n    at new Promise (<anonymous>)\n    at /opt/outline/node_modules/@outlinewiki/koa-passport/lib/framework/koa.js:193:12\n    at /opt/outline/node_modules/@outlinewiki/koa-passport/lib/framework/koa.js:143:7\n    at new Promise (<anonymous>)\n    at passportAuthenticate (/opt/outline/node_modules/@outlinewiki/koa-passport/lib/framework/koa.js:107:15)\n    at passportMiddleware (/opt/outline/build/server/middlewares/passport.js:68:7)\n    at dispatch (/opt/outline/node_modules/koa-compose/index.js:44:32)\n    at next (/opt/outline/node_modules/koa-compose/index.js:45:18)\n    at /opt/outline/node_modules/koa-router/lib/router.js:321:16\n    at dispatch (/opt/outline/node_modules/koa-compose/index.js:44:32)\n    at next (/opt/outline/node_modules/koa-compose/index.js:45:18)\n    at methodOverrideMiddleware (/opt/outline/build/server/middlewares/methodOverride.js:22:12)\n    at dispatch (/opt/outline/node_modules/koa-compose/index.js:44:32)\n    at next (/opt/outline/node_modules/koa-compose/index.js:45:18)\n    at /opt/outline/node_modules/koa-router/lib/router.js:321:16\n    at dispatch (/opt/outline/node_modules/koa-compose/index.js:44:32)\n    at next (/opt/outline/node_modules/koa-compose/index.js:45:18)\n    at /opt/outline/node_modules/@outlinewiki/koa-passport/lib/framework/koa.js:60:14","level":"error","message":"Error during authentication"}`

**Reply by tommoor** [reply]
There is more information in the `AuthorizationError` that isn't contained in the logs, the easiest way to see that would be to hook up [Sentry](https://sentry.io/) for error reporting. Just need to create an account and then add the `SENTRY_DSN` to your environment, this should expose more.
